pybop/models/lithium_ion/base_echem.py

```diff
@@ -213,30 +215,31 @@ def cell_mass(self, parameter_set: Optional[ParameterSet] = None):
         parameter_set = parameter_set or self._parameter_set
 
         def mass_density(
-            active_material_vol_frac, density, porosity, electrolyte_density
+            active_material_vol_frac, density, porosity, electrolyte_density, cb_density
```

Comment:
Perhaps `rho_carbon_black` or `carbon_black_density` would be more descriptive here? Took me a second to comprehend `cb_density` was the carbon black density.